examples/model_compress/pruning/configure_example.yaml

```diff
@@ -1,9 +0,0 @@
-AGPruner: 
```

Comment:
what is this file used for?

Reply:
no used in the examples, looks redundant, so remove it.